### PR TITLE
[Fix] TAO 9843 - Media interaction playback continues after leaving page

### PR DESCRIPTION
**Pull request by AndreyNegrov**
**Related to task:**  https://oat-sa.atlassian.net/browse/TAO-9843
**Description:** Media interaction: playback continues after leaving page if autostart is set
**How to test:** 
- Create an item with Media interaction
- Upload audio/video file
- Set 'Autostart' in the Interaction Properties
- Save and Preview

**Expected result:** playback autostarted in authoring mode finishes as soon as Preview mode is opened.

### CODE REVIEW

**Review by dresha-dev — APPROVED** [review]
⚠️ The issue still exist when user is opening preview mode and interaction is "in active" state.

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

File: views/js/qtiCreator/renderers/interactions/MediaInteraction.js [hunk 1]
```diff
@@ -46,6 +46,10 @@ define([
             this.getOption('responseOptionForm'),//note : no response required...
             options
         );
+
+        $(document).on('open-preview.qti-item', () => {
+            interaction.mediaElement.execute('pause');
```
Comment (zagovorichev) [inline views/js/qtiCreator/renderers/interactions/MediaInteraction.js:51]: I would use `stop` instead

**Review by zagovorichev — CHANGES_REQUESTED** [review]
There is an issue in the behavior: on the second or third preview action, it doesn't pause the video

File: views/js/qtiCreator/renderers/interactions/MediaInteraction.js (hunk 1 above)
Comment (AndreyNegrov) [reply views/js/qtiCreator/renderers/interactions/MediaInteraction.js]: fixed

**Comment by AndreyNegrov** [comment]
> There is an issue in the behavior: on the second or third preview action, it doesn't pause the video

fixed

**Comment by habemuscode** [comment]
Merge develop and bump manifest again

**Review by habemuscode — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Pull request merged**